README.adoc

```diff
@@ -552,6 +552,33 @@ validates_length_of :email, maximum: 100
 validates :email, presence: true, length: { maximum: 100 }
 ----
 
+=== Custom Validation Methods
+
+When naming custom validation methods, adhere to the simple rules:
+
+ - `validate :method_name` reads like a natural statement
+ - the method name explains what it checks
+ - the method is recognizable as a validation method by its name, not a predicate method
+
+[source,ruby]
+----
+# good
+validate :expiration_date_cannot_be_in_the_past
+validate :discount_cannot_be_greater_than_total_value
+validate :ensure_same_topic
+validate :referrer_valid
```

Comment:
This might not be a good example - it sounds like a predicate method. The others make sense to me though. Maybe we remove this one line?

Reply:
Thanks for the heads-up! I was under the impression that I've removed this one in the last proofreading sweep.